### DISCUSSION TITLE
Set Spacial Discretization scheme using settings objects

### DISCUSSION THREAD

**Discussion by gibbsycfd** [ansys/pyfluent#1145] [Q&A] [answered]
I'm trying to set the momentum discretization term to first order in fluent using settings objects, unsure what follows the text below.

root.solution.methods.discretization_scheme.

**Comment by gibbsycfd** [comment, marked as answer]
root.solution.methods.discretization_scheme["mom"] = "First Order Upwind"

**Comment by dnwillia-work** [comment]
I think if you run this:

`root.solution.methods.discretization_scheme["mom"].allowed_values()`

then it will tell you the acceptable string values.
